src/core/opamCoreConfig.ml

```diff
@@ -157,11 +159,19 @@ let initk k =
         | true -> Some `Extended
         | false -> None)
     ) in
+  let answer =
+    match E.confirmlevel (), E.yes (), E.no () with
+    | Some c, _,  _ -> Some c
+    | _, Some true, _ -> Some `all_yes
+    | _, _, Some true -> Some `all_no
+    | _ -> None
+(*
```

Comment:
priority over environment variables: confim level > yes > no

Reply:
It's worth having that in the docstrings too, I think.